src/Features/Core/Portable/QuickInfo/CommonSemanticQuickInfoProvider.cs

```diff
@@ -214,49 +208,35 @@ void AddSection(string kind, ImmutableArray<TaggedText> taggedParts)
                 symbol = symbol.ContainingType;
             }
 
-            var documentationContent = GetDocumentationContent(symbol, groups, semanticModel, token, formatter, cancellationToken);
-
-            if (!documentationContent.IsDefaultOrEmpty)
-            {
-                AddSection(QuickInfoSectionKinds.DocumentationComments, documentationContent);
-            }
+            if (groups.TryGetValue(SymbolDescriptionGroups.Documentation, out var docParts) && !docParts.IsDefaultOrEmpty)
+                AddSection(QuickInfoSectionKinds.DocumentationComments, docParts);
 
-            var remarksDocumentationContent = GetRemarksDocumentationContent(workspace, symbol, groups, semanticModel, token, formatter, cancellationToken);
-            if (!remarksDocumentationContent.IsDefaultOrEmpty)
+            var remarksDocumentation = GetRemarksDocumentationContent(workspace, groups, semanticModel);
```

Comment:
nit: might as well just inline this since it's a very simple method and to be consistent with getting ReturnsDocumentation below.

Reply:
sure.